.github/workflows/run_comparative_benchmark.yml

```diff
@@ -13,6 +13,7 @@ on:
     # Scheduled to run at 09:00 UTC and 21:00 UTC.
     - cron: '0 09,21 * * *'
   workflow_dispatch:
+  pull_request:
```

Comment:
For testing, you can comment out unrelated steps (e.g. JAX-XLA, XLA-HLO) to save some resources.

Reply:
Will do, thanks!